code/game/area/LV759_Hybrisa_Prospera.dm

```diff
@@ -0,0 +1,939 @@
+	name = "Nova Medica Hospital Complex - Outgoing Ward"
+/area/lv759/indoors/hospital/central_hallway
+	name = "Nova Medica Hospital Complex - Central Hallway"
+/area/lv759/indoors/hospital/east_hallway
+	name = "Nova Medica Hospital Complex - East Hallway"
+/area/lv759/indoors/hospital/medical_storage
+	name = "Nova Medica Hospital Complex - Medical Storage"
+/area/lv759/indoors/hospital/operation
+	name = "Nova Medica Hospital Complex - Operation Theatres & Observation"
+/area/lv759/indoors/hospital/patient_ward
+	name = "Nova Medica Hospital Complex - Patient Ward"
+/area/lv759/indoors/hospital/virology
+	name = "Nova Medica Hospital Complex - Virology"
+/area/lv759/indoors/hospital/morgue
+	name = "Nova Medica Hospital Complex - Morgue"
+/area/lv759/indoors/hospital/icu
+	name = "Nova Medica Hospital Complex - Intensive Care Ward"
+/area/lv759/indoors/hospital/storage
+	name = "Nova Medica Hospital Complex - Storage Room"
+/area/lv759/indoors/hospital/maintenance_north
+	name = "Nova Medica Hospital Complex - Maintenance North"
+/area/lv759/indoors/hospital/maintenance_south
+	name = "Nova Medica Hospital Complex - Maintenance South"
+/area/lv759/indoors/hospital/janitor
+	name = "Nova Medica Hospital Complex - Janitors Closet"
+
+// Mining
+
+/area/lv759/indoors/mining_outpost
+	icon_state = "mining"
+	minimap_color = MINIMAP_AREA_MINING
+/area/lv759/indoors/mining_outpost/north
+	name = "KMCC - Mining Outpost - North"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/north_maint
+	name = "KMCC - Mining Outpost - North - Maintenance"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/northeast
+	name = "KMCC - Mining Outpost - Northeast"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/south
+	name = "KMCC - Mining Outpost - South"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/vehicledeployment
+	name = "KMCC - Mining Vehicle Deployment South"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/processing
+	name = "KMCC - Mining Processing & Storage"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/east
+	name = "KMCC - Mining Outpost - East"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/east_dorms
+	name = "KMCC - Mining Outpost - East Dorms"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/east_deploymentbay
+	name = "KMCC - Mining Outpost - East - Deployment Bay"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/east_command
+	name = "KMCC - Mining Outpost - East - Command Center"
+	icon_state = "mining"
+/area/lv759/indoors/mining_outpost/cargo_maint
+	name = "KMCC - Mining Outpost - East - Maintenance"
+	icon_state = "mining"
+
+// Electrical Substations
+
+/area/lv759/indoors/electical_systems
+	minimap_color = MINIMAP_AREA_COLONY_ENGINEERING
+/area/lv759/indoors/electical_systems/substation1
+	name = "Electrical Systems - Substation One - Control Room"
+	icon_state = "power0"
+/area/lv759/indoors/electical_systems/substation2
```

Comment:
You maybe should consider adding `is_landing_zone = TRUE` to this area. Unless you shrink the LZ2 area to not cut off this area as much.

Reply:
Everything regarding the LZ's should hopefully be good now.